pyproject.toml

```diff
@@ -23,10 +23,12 @@ requires = ["poetry_core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+astroid = "^2.15.2"
+python = "^3.7.2"
```

Comment:
Could the `.2` part be removed?

Reply:
Unfortunately no. Minimal python version for `astroid` is `3.7.2`